### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

File: src/main/java/org/jboss/pnc/buildkitchen/model/Artifact.java [hunk 1]
```diff
@@ -18,27 +18,27 @@
...
 @Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"purl", "sha256"}))
 public class Artifact extends PanacheEntity {
 
     @NotNull
-    @Column(unique = true)
     public String purl;
 
+    @NotNull
+    public String sha256;
```
Comment (vibe13) [inline src/main/java/org/jboss/pnc/buildkitchen/model/Artifact.java:43]: Should this be added to the ArtifactDTO as well?

File: src/main/java/org/jboss/pnc/buildkitchen/api/PNCBuild.java
Comment (vibe13) [inline src/main/java/org/jboss/pnc/buildkitchen/api/PNCBuild.java:1]: Should this be named PNCBuildDTO?

**Review by vibe13 — CHANGES_REQUESTED** [review]

File: src/main/java/org/jboss/pnc/buildkitchen/api/PNCBuild.java
Comment (vibe13) [reply src/main/java/org/jboss/pnc/buildkitchen/api/PNCBuild.java:1]: Ok I have seen the JIRA comment on this. Never mind.

File: src/main/java/org/jboss/pnc/buildkitchen/model/Artifact.java [hunk 2]
```diff
@@ -18,40 +18,60 @@
...
-                        """,
-                Parameters.with("scmInfo", purls)).<Artifact>stream()
-                .collect(Collectors.toMap(a -> a.purl, Function.identity()));
+    public static Map<PurlSha, Artifact> findByPurls(Set<PurlSha> purls) {
+        List<PurlSha> purlShas = new ArrayList<>(purls);
+        String query = "FROM Artifact a WHERE ";
+        Map<String, String> params = new HashMap<>();
+        for (int i = 0; i< purlShas.size(); i++){
+            if(i > 0) {
+                query += " AND ";
```
Comment (vibe13) [inline src/main/java/org/jboss/pnc/buildkitchen/model/Artifact.java]: Is it an OR?

**Review by vibe13 — CHANGES_REQUESTED** [review]

File: src/main/java/org/jboss/pnc/buildkitchen/model/Artifact.java (hunk 2 above)
Comment (janinko) [reply src/main/java/org/jboss/pnc/buildkitchen/model/Artifact.java]: yes, it should

**Review by vibe13 — APPROVED** [review]

**Pull request merged**